docs/src/views/FAQ.tsx

```diff
@@ -12,6 +12,35 @@ const Answer = props => <p>{props.content}</p>
 
 export default () => (
   <DocPage title="FAQ">
+    <Header as="h2" content="Content" />
+    <ul>
+      <li>
+        {link('General', '#General')}
+        <ul>
+          <li>{link('Does Stardust support mobile?', '#does-stardust-support-mobile')}</li>
+          <li>
+            {link(
+              "How can @mixins be used in the Stardust's styles?",
+              '#how-can-mixins-be-used-in-the-stardusts-styles',
+            )}
```

Comment:
Maybe we can add here an example, to be more clear for users.

Reply:
Can you suggest?

Reply:
We can reference the `getBorderFocusStyles` function that we are already using in the button styles in Teams theme.